src/components/Loading/Loading.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react'
+import styles from './Loading.module.scss';
+import logo from '../../assets/Temp_Roadwatch_logo.svg';
+// import logo2 from '../../assets/Temp_Roadwatch_logo-2.svg'
```

Comment:
Not sure what this is, but remove anything you don't need

Reply:
The excess import statements were removed. I was testing for both logos at the time.